src/linux/xt_coova.c

```diff
@@ -467,7 +467,9 @@ static int coova_seq_open(struct inode *inode, struct file *file)
 	if (st == NULL)
 		return -ENOMEM;
 
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(3,10,0)
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(5,17,0)
+	st->table = pde_data(inode);
+#elif LINUX_VERSION_CODE >= KERNEL_VERSION(3,10,0)
```

Comment:
This needs an upper bound (before 5.17.0?)

Reply:
Thanks for taking a look sevan.
I thought the upper bound would be guarded by the else in `#elif`, so any kernel at or after 5.17 would not reach this?

Cheers

Reply:
it's more to protect against future edits, rather than compiling/parsing (at a glance to the human they both apply to 5.17, one just more so :) )